docs/modules/ROOT/pages/documentation/user-guides/starter-with-jpa.adoc

```diff
@@ -0,0 +1,101 @@
+[[starter-with-jpa]]
+= Add JPA to MicroProfile Starter generated project
+
+This guide shows the steps to add JPA support to an application generated by the MicroProfile Starter project for Payara Micro.
+
+== MySQL based application with an external definition of the DataSource.
+
+1) Start by generating a project using the MicroProfile Starter website. You can use the Command Line to create a project for Payara Micro without any examples of the MicroProfile specifications.
```

Comment:
I would add a link to the MicroProfile Starter website. Not everybody can use curl on their machine.